### PR TITLE
tageditor: init at 3.3.10, tagparser: init at 9.4.0

### LINKED ISSUE #107006: Tag Editor
**Project description**
A tag editor with Qt GUI and command-line interface. Supports MP4 (iTunes), ID3, Vorbis, Opus, FLAC and Matroska.

**Metadata**

* homepage URL: https://github.com/Martchus/tageditor
* source URL: https://github.com/Martchus/tageditor
* license: mit, GPLv2.0
* platforms: linux

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Closes #107006

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

CC @luxferresum for testing.

### CODE REVIEW

**Comment by taku0** [comment]
Please rebase on `master`.

**Comment by matthiasbeyer** [comment]
Done

**Comment by taku0** [comment]
It runs fine.

Please replace `stdenv.lib` with `pkgs.lib`.

> Warning: `stdenv.lib` is deprecated and will be removed in the next release. Please use `pkgs.lib` instead. For more information see https://github.com/NixOS/nixpkgs/issues/108938

**Comment by matthiasbeyer** [comment]
Done. If you approve, I'll squash.

File: pkgs/development/libraries/tagparser/default.nix [hunk 1]
```diff
@@ -22,7 +23,7 @@ stdenv.mkDerivation rec {
     cpp-utilities zlib
   ];
 
-  meta = with stdenv.lib; {
+  meta = with pkgg.lib; {
```
Comment (taku0) [inline pkgs/development/libraries/tagparser/default.nix:26]: `pkgg` → `pkg`.

**Review by taku0 — APPROVED** [review]
Please squash.

**Comment by taku0** [comment]
Waiting for ofborg...

**Comment by taku0** [comment]
The pending check is the tests on `x86_64-darwin` while this package is Linux only. Merging.

**Pull request merged**